### DISCUSSION TITLE
Quickstart Guide - Having issues.

### DISCUSSION THREAD

**Discussion by alingham** [gregsullivan/_tw#97] [Q&A] [answered]
Hi team,

I've followed the instructions in the quickstart guide, but unfortunately they're not entirely clear and I'm having issues.

I've made the theme package easy enough. Let's call it tailwindtheme. I've opened tailwindtheme.zip and extracted the files in it to "wp-content/themes" as the guide states.
I am ASSUMING that when it says "Move this folder to......." that the THIS folder you're referring to is the folder created when tailwindtheme is extracted. Right? Or do you mean the "theme" folder inside the tailwindtheme folder? That part is not very clear...

`
"Move this folder to wp-content/themes in your local development environment"
`
Moving on, I now have (locally) wp-content/themes/tailwindtheme/
which has the folders "javascript", "node_modules", "node_scripts", "tailwind" and "theme" inside it.

Inside that I run the npm commands to get it running, no issues.
But then the quick start guide says 
`Activate this theme in WordPress`

Now... 
Nowhere did it say to upload it to Wordpress, except for later on.
But I uploaded the tailwindtheme folder to my server, which has the Wordpress site on it and went to activate the theme in Wordpress as expected, except that it comes back saying: Broken Theme: Stylesheet is missing.

So I'm a bit stuck and there's very little documentation or forums around this. The quickstart guide needs to be a little clearer as to which folder you mean by "Move this folder..." as well as what you mean when you say "Activate this theme in Wordpress". If you're meaning a local environment build of Wordpress, then it might be worth stating that as not everyone has one running.

If anyone can point me in the right direction to getting this sorted so I can begin developing my theme, that'd be great as well.

Thanks

Al.

**Comment by gregsullivan** [comment, marked as answer]
Hi Al,

Thanks for your post! I'm always interested in hearing about ways to make the documentation clearer, so this is great. Here is the old quickstart for Installation:

1. [Generate your theme](https://underscoretw.com/)
2. Unzip the theme archive
3. Move the resulting folder to wp-content/themes in your local development environment
4. Run npm install && npm run dev in the moved folder
5. Activate the theme in WordPress

I've revised it as follows on the [root documentation page](https://underscoretw.com/docs/):

1. [Generate your theme](https://underscoretw.com/)
2. Unzip the archive
3. Move the entire unzipped folder to wp-content/themes in your local WordPress installation
4. Run npm install && npm run dev in the moved folder
5. Activate the theme in your local WordPress installation

Does that phrasing answer all of your questions and make things clearer? If it does, I'll update the quickstart everywhere it appears. If not, I'd love to make further improvements.

Thanks again!

**Reply by alingham** [reply]
Looks good. That makes more sense. 
If someone doesn't have a local Wordpress installation, is there a method we should use?

**Reply by gregsullivan** [reply]
Great, thank you! I'll roll this out in more places moving forward.

I would strongly suggest using a local WordPress development environment. If you didn't want to do that, you could theoretically install Node on a server and work there. You could also use something like Github Codespaces, though I'm not sure what the state of the art is in terms of WordPress via Codespaces.
